TombEngine/Specific/configuration.h

```diff
@@ -36,15 +36,24 @@ constexpr auto REGKEY_ENABLE_AUTO_TARGETING	   = "EnableAutoTargeting";
 constexpr auto REGKEY_ENABLE_RUMBLE			   = "EnableRumble";
 constexpr auto REGKEY_ENABLE_THUMBSTICK_CAMERA = "EnableThumbstickCamera";
 
+// Input keys
+constexpr auto REGKEY_MOUSE_SENSITIVITY = "MouseSensitivity";
+constexpr auto REGKEY_MOUSE_SMOOTHING	= "MouseSmoothing";
+
 struct GameConfiguration 
 {
+	static constexpr auto DEFAULT_SHADOW_MAP_SIZE	= 1024;
+	static constexpr auto DEFAULT_SHADOW_BLOBS_MAX	= 16;
+	static constexpr auto DEFAULT_MOUSE_SENSITIVITY = 6;
+	static constexpr auto DEFAULT_MOUSE_SMOOTHING	= 1;
+
 	// Graphics
 	int		   ScreenWidth		  = 0;
 	int		   ScreenHeight		  = 0;
 	bool	   EnableWindowedMode = false;
 	ShadowMode ShadowType		  = ShadowMode::None;
-	int		   ShadowMapSize	  = 1024;
-	int		   ShadowBlobsMax	  = 16;
+	int		   ShadowMapSize	  = DEFAULT_SHADOW_MAP_SIZE;
```

Comment:
I'm not sure if this is gonna affect, but I'm seeing in this header code (configuration.h) DEFAULT_SHADOW_MAP_SIZE has the value of 1024, however in the source code (configuration.cpp) was taken to replace the shadowMapSize constant that was 512.

Was an intentional change? or would be okey having 512 changed to 1024? Have not tested yet.

Reply:
Yeah, I think there's not gonna be problem with that, I pointed that just to be aware.